tools/convert_from_16p2

```diff
@@ -66,14 +66,14 @@ table shows the ffmpeg pixel_format name for each of these formats:
 
     args = parser.parse_args()
 
-    f = open(args.infilename, "r")
+    f = open(args.infilename, "rb")
 
     outfilename = args.infilename + ".yuv"
 
     data = f.read()
```

Comment:
Under Python 2.x, the change below will fail because the input is still read into a `str`. Wrapping the read data in a byte array gives the indexing behaviour below.

```suggestion
    data = bytearray(f.read())
```